src/modules/quizzes.js

```diff
@@ -140,10 +154,13 @@ class Quizzes {
    * @function getQuizResults
    * @description Retrieve quiz recommendation and filter expression from Constructor.io API
    * @param {string} id - The identifier of the quiz
-   * @param {string} [parameters] - Additional parameters to refine result set
+   * @param {string} parameters - Additional parameters to refine result set
+   * @param {array} parameters.answers - An array of answers in the format [[1,2],[1]]
    * @param {string} [parameters.section] - Product catalog section
-   * @param {array} [parameters.answers] - An array of answers in the format [[1,2],[1]]
    * @param {string} [parameters.versionId] - Specific version identifier for the quiz
+   * @param {number} [parameters.page] - The page number of the results
+   * @param {number} [parameters.resultsPerPage] - The number of results per page to return
+   * @param {object} [parameters.filters] - Key / value mapping (dictionary) of filters used to refine results
```

Comment:
we're missing the quiz_session_id as a parameter

Reply:
Handled here https://github.com/Constructor-io/constructorio-client-javascript/pull/245/